mariadb_kernel/maria_magics/load.py

```diff
@@ -0,0 +1,72 @@
+"""This class implements the %load magic command"""
+
+help_text = """
+The %load magic command has the following syntax:
+    > %load [csv file path] [table name] [skip row num](optional)
```

Comment:
%load csv_file_path table_name [skip_row_num]
This way it should be implicit that skip_row_num is optional

Reply:
I think the convention is `%load <csv file path> <table name> [skip rows]`
Maybe there should be a way to specify the ending row number as well, either `[from] [to]` or `[from] [number of rows]`.

Reply:
I don't understand what you mean, sorry. What do you mean with convention? For what?

Reply:
Usually, mandatory arguments are enclosed in angle brackets `<argument>`, while optional arguments are enclosed in square brackets (`[argument]`).

Reply:
Got it now, thanks! MariaDB KB docs have regular name for mandatory args and [] for optional ones. Whilst it's not necessarily my intent to be consistent with SQL documentation in MariaDB, I vote for csv_file_path table_name [skip_rows], seems more intuitive to me.